go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/dustin/go-humanize
+
+go 1.16
```

Comment:
That doesn't mean that only 1.16+ is supported: https://golang.org/ref/mod#go-mod-file-go